index.md

```diff
@@ -8,15 +8,60 @@ redirect_from:
 
 Our Networks 2019 explores the collective care and maintenance of alternative networking practices&mdash;new protocols, peer-to-peer connections, offline-first computing, or community-based governance and ownership. Building on last year, where we asked how to move beyond solitary practices of do it yourself (DIY), in 2019 we explore what critical and creative practices offer **RE: Infrastructures** of alternatives.
 
-How can these emerging **Infrastructures of <span id="word"></span>** provide spaces to imagine<span class="word-spacer"></span>  
+How can these emerging **Infrastructures of <span id="word"></span>** provide spaces to imagine<span class="word-spacer"></span>
 radically different futures, resist easy co-option by the forces of late-stage capitalism, and work in solidarity as plural networks seeking emancipatory change? To answer this we draw on insights from organizers, technologists, artists, and scholars to ensure the focus is not only on the technical aspects of infrastructure, but also the social and cultural. Their diverse perspectives suggest ways we can make space for livelihoods, solidarity, and survival; draw on history and place to bring our fullest imaginations to this moment; and open ways to think these questions together as we build transitional forms.
 
 {% include 2019-quotes.html %}
 
-## Updates
+We are inviting proposals for talks, workshops, discussions, demonstrations, art pieces, and interventions to explore this theme. Topics could include, but are in no way limited to:
+
+- Practices of reparative networking
+- Remediation practices for communication infrastructures
+- Uncooptable computing reconfigurations
+- New relations while refusing bad ones
```

Comment:
Not sure about "New relations while refusing bad ones" maybe "bad ones" seems a bit vague? Otherwise all LGTM

Reply:
unfair, exploitative, unsustainable, asymmetric, unwelcomed

Reply:
Reconstructing technology beyond refusal

Reply:
Reconstructing technology after refusal

Reply:
Technology relations after refusal